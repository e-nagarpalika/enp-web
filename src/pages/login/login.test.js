/** @format */

import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import Login from "./login";

test("renders a message", () => {
  const { container } = render(<Login />);

  expect(container).toMatchSnapshot();
});
