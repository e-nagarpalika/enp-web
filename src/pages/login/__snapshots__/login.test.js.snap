// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div>
  <section
    class="sc-bdnxRM emIdNA"
  >
    <div
      class="card"
    >
      <div
        class="card-body "
      >
        <h5
          class="card-title"
        >
          Log In
        </h5>
        <form
          style="display: flex; justify-content: center;"
        >
          <input
            id="recaptcha-container"
            type="hidden"
          />
          <div
            class="mb-3 "
          >
            <label
              class="form-label"
              for="phoneNumber"
            >
              Phone Number
            </label>
            <input
              aria-describedby="phoneNumberHelp"
              class="form-control"
              id="phoneNumber"
              placeholder="Phone Number"
              type="text"
            />
            <div
              class="form-text"
              id="phoneNumberHelp"
            >
              We'll never share your phone number with anyone else.
            </div>
          </div>
          <button
            class="btn btn-primary"
            style="align-self: flex-end;"
            type="submit"
          >
            Submit
          </button>
        </form>
      </div>
    </div>
  </section>
</div>
`;
